editor/store/actions.js

```diff
@@ -359,6 +359,19 @@ export function setActivePanel( panel ) {
 	};
 }
 
+/**
+ * Returns an action object usid in signalling that the user switched the active plugin.
+ * 
+ * @param  {String} plugin  The plugin name
+ * @return {Object}         Action object
+ */
+export function setActivePlugin( plugin ) {
```

Comment:
setActive(Plugin)Sidebar might be a better name, because a plugin can have multiple sidebars.